.eslintrc.js

```diff
@@ -1,4 +1,5 @@
 const rules = {
+  "@typescript-eslint/no-floating-promises":"off"
```

Comment:
Could you please explain why you disable the `no-floating-promises`?

I believe it is very helpful when we coding and missed the `await` in the `if` closure.